images/java/Dockerfile.ubuntu

```diff
@@ -0,0 +1,10 @@
+FROM codercom/enterprise-base:ubuntu
+
+# Run everything as root
+USER root
+
+# Install JDK (OpenJDK 14)
+RUN DEBIAN_FRONTEND="noninteractive" apt-get install -y openjdk-14-jdk
```

Comment:
Also - it seems that OpenJDK 11 is the most commonly used Java version nowadays, after Java 8. In fact, IntelliJ IDEs come with a packaged OpenJDK JRE 11.

Reply:
Ah yeah no kidding, looks like Java 8 is still ~65% of market share, I'll just switch it to use that.